gcc/config/riscv/riscv-builtins.cc

```diff
@@ -120,6 +131,18 @@ AVAIL (crypto_zksh32, TARGET_ZKSH && !TARGET_64BIT)
 AVAIL (crypto_zksh64, TARGET_ZKSH && TARGET_64BIT)
 AVAIL (crypto_zksed32, TARGET_ZKSED && !TARGET_64BIT)
 AVAIL (crypto_zksed64, TARGET_ZKSED && TARGET_64BIT)
+
+/* p ext */
+AVAIL (zpn, TARGET_ZPN)
+AVAIL (zpn64, TARGET_ZPN &&TARGET_64BIT)
+AVAIL (zpn32, TARGET_ZPN && !TARGET_64BIT)
+AVAIL (zpsfoperand, TARGET_ZPSFOPERAND)
+AVAIL (zpsfoperand32, TARGET_ZPSFOPERAND && !TARGET_64BIT)
+AVAIL (zpsfoperand64, TARGET_ZPSFOPERAND &&TARGET_64BIT)
+AVAIL (zbpbo, TARGET_ZBPBO)
+AVAIL (zbpbo32, TARGET_ZBPBO && !TARGET_64BIT)
+AVAIL (zbpbo64, TARGET_ZBPBO &&TARGET_64BIT)
+
```

Comment:
A uniform format is recommended here. For example, “AVAIL (zpn64, TARGET_ZPN && TARGET_64BIT)”

Reply:
ok , clang-format action missing...

Reply:
ok, `clang-format` action missing...